packages/blockchain/src/consensus/clique.ts

```diff
@@ -0,0 +1,557 @@
+import { debug as createDebugLogger } from 'debug'
+import { Block, BlockHeader } from '@ethereumjs/block'
+import { ConsensusAlgorithm } from '@ethereumjs/common'
+import { Address, rlp, bigIntToBuffer, bufferToBigInt } from 'ethereumjs-util'
+import Blockchain from '..'
+import { Consensus, ConsensusOptions } from './interface'
+
+const debug = createDebugLogger('blockchain:clique')
+
+// Clique Signer State: [blockNumber, signers]
+type CliqueSignerState = [bigint, Address[]]
```

Comment:
our TS version has named tuples now, so you can inline this comment and the ones below like this

```suggestion
// Clique Signer State
type CliqueSignerState = [blockNumber: bigint, signers: Address[]]
```

Reply:
Ah nice, I didn't know that. Updated